packages/astro/test/static-build.test.js

```diff
@@ -6,6 +6,10 @@ function addLeadingSlash(path) {
 	return path.startsWith('/') ? path : '/' + path;
 }
 
+function removeBasePath(path) {
+	return path.replace('/subpath', '')
+}
+
```

Comment:
I was about to ask if this was needed, but I see it's just for the test.

Reply:
👍 I was going to move `/subpath` out to a variable, but it's hardcoded in the fixture's config so I didn't bother.  I'll add a comment to make that more clear